### ISSUE TITLE
[Proposal] UI implementation guideline for a11y

### ISSUE THREAD

**Issue by fumiya-kume** [DroidKaigi/conference-app-2023#467]
**Idea Description**

Currently bunch of UI missing a11y supporting, It's not suitable situation for the handicapped attendees as you know.
From this situation, kind of guideline for a11y implementation to keep a11y supporting quality over the contributors.

For expand the knowledge of a11y (for Android developers industry as well 😃 ), it's nice to have kind of those document.   

# Proposal solution

1. Make guideline at `Readme.md` or reuse the Google's document
2. Add section at `CONTRIBUTING.md` 
3. (nice to have) Enable automation tests to checking on each PR

For Item1, This page(https://developer.android.com/guide/topics/ui/accessibility) will be basis of discussion for document.
For Item3, We can refer the [document](https://developer.android.com/guide/topics/ui/accessibility/testing?hl=ja&continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fmake-your-android-app-accessible%3Fhl%3Dja%23article-https%3A%2F%2Fdeveloper.android.com%2Fguide%2Ftopics%2Fui%2Faccessibility%2Ftesting#espresso)

# Discussion points

1. Contributor's cost/ too much requirements
2. Reuse Google's material or in-house documentation for the issue.

**Reference images and links**

**Comment by takahirom** [comment]
Thanks for your proposal!

> 1. Make guidelines in Readme.md or reuse Google's document
2. Add a section to CONTRIBUTING.md
3. (nice to have) Enable automation tests to check on each PR

1. Linking to Google's document instead of copying it might be a good idea, as it will always be up-to-date. However, in that case, it might not be viewed as often. Probably, dealing with the document might be challenging, so I'd like to consider option 3.
2. For the code, I'd prefer to gather it in a place other than CONTRIBUTING.md.
3. Regarding 3, Roborazzi's functionality to visualize accessibility status and take screenshots of it might be helpful. It allows us to develop with constant awareness of accessibility. https://github.com/DroidKaigi/conference-app-2023/issues/444

**Comment by takahirom** [comment]
I feel like there's nothing we can do, so let me close this. If there's anything you want to do, please feel free to reach out to me!
